api/handlers/attributes.go

```diff
@@ -0,0 +1,295 @@
+package handlers
+
+import (
+	"fmt"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/equinor/vds-slice/internal/cache"
+	"github.com/equinor/vds-slice/internal/core"
+)
+
+// AttributesAlongSurfacePost godoc
+// @Summary  Returns horizon attributes along the surface
+// @description.markdown attribute_along
+// @Tags     attributes
+// @Param    body  body  AttributeAlongSurfaceRequest  True  "Request Parameters"
+// @Accept   application/json
+// @Produce  multipart/mixed
+// @Success  200 {object} core.AttributeMetadata "(Example below only for metadata part)"
+// @Failure  400 {object} ErrorResponse "Request is invalid"
+// @Failure  500 {object} ErrorResponse "openvds failed to process the request"
+// @Router   /attributes/surface/along  [post]
+func (e *Endpoint) AttributesAlongSurfacePost(ctx *gin.Context) {
+	var request AttributeAlongSurfaceRequest
+	err := parsePostRequest(ctx, &request)
+	if abortOnError(ctx, err) {
+		return
+	}
+
+	e.makeDataRequest(ctx, request)
+}
+
+// AttributesBetweenSurfacesPost godoc
+// @Summary  Returns horizon attributes between provided surfaces
+// @description.markdown attribute_between
+// @Tags     attributes
+// @Param    body  body  AttributeBetweenSurfacesRequest  True  "Request Parameters"
+// @Accept   application/json
+// @Produce  multipart/mixed
+// @Success  200 {object} core.AttributeMetadata "(Example below only for metadata part)"
+// @Failure  400 {object} ErrorResponse "Request is invalid"
+// @Failure  500 {object} ErrorResponse "openvds failed to process the request"
+// @Router   /attributes/surface/between  [post]
+func (e *Endpoint) AttributesBetweenSurfacesPost(ctx *gin.Context) {
+	var request AttributeBetweenSurfacesRequest
+	err := parsePostRequest(ctx, &request)
+	if abortOnError(ctx, err) {
+		return
+	}
+
+	e.makeDataRequest(ctx, request)
+}
+
+func (request AttributeAlongSurfaceRequest) execute(
+	handle core.DSHandle,
+) (data [][]byte, metadata []byte, err error) {
+	err = validateVerticalWindow(request.Above, request.Below, request.Stepsize)
+	if err != nil {
+		return
+	}
+
+	interpolation, err := core.GetInterpolationMethod(request.Interpolation)
+	if err != nil {
+		return
+	}
+
+	metadata, err = handle.GetAttributeMetadata(request.Surface.Values)
+	if err != nil {
+		return
+	}
+
+	data, err = handle.GetAttributesAlongSurface(
+		request.Surface,
+		request.Above,
+		request.Below,
+		request.Stepsize,
+		request.Attributes,
+		interpolation,
+	)
+	if err != nil {
+		return
+	}
+
+	return data, metadata, nil
+}
+
+func (request AttributeBetweenSurfacesRequest) execute(
```

Comment:
Why such weird positioning of execute methods for both Attribute Requests?
Shouldn't they be at the end of their respective class definitions, like in other files??